### ISSUE TITLE
Create class for household assets

### ISSUE THREAD

**Issue by aellenhicks** [ufbmi/OMRSE#67]
Proposed definition by zhengj2007 in issue #62 

household assets: An information content entity about items that a household owns such as a car, funiture, etc. (parent: IAO_0000030 information content entity)

**Comment by aellenhicks** [comment]
How about the prosed label and definition?
'household asset data item' - A socio-economic data item about the possessions of a household that have monetary value.

If data item is too specific, we can change to information content entity.

**Comment by cstoeckert** [comment]
Yes this works (see related https://github.com/ufbmi/OMRSE/issues/66) comment about assets.
